movie.sh

```diff
@@ -1,6 +1,48 @@
 #!/bin/bash
-echo 'please input the movie name'
-read name
-echo $name
-var=$(curl -s http://www.omdbapi.com/\?i\=\&t\=$name | jq -r '.imdbRating')
-echo $var
+#Get command line argument
+if [ -z $1 ];
+then 
+    echo 'Directory not provided. Searching in current directory'
+    dir='.'
+else 
+    echo 'Input Directory is' $1
+    dir=$1
+fi
+
+#Get absolute path from the input directory
+path=$(find ~/ -name $dir)
```

Comment:
I guess this will cause some issues if the specified directory exists at multiple paths.